src/frontend/console/index.js

```diff
@@ -0,0 +1,35 @@
+define(['jquery'], function(){
+
+    var last_type = -1;
+
+    function addToConsole(type, text){
+            $('.jumbotron').append('<br />');
```

Comment:
it's not the best idea to refer to this element via .jumbotron. Please give it a name like #console-output